app/views/manage_collections/show.html.erb

```diff
@@ -22,7 +22,8 @@
                   options_for_select(@templates),
                   prompt: 'Select Template Name',
                   class: 'template-select',
-                  style: 'display:none;'
+                  style: 'display:none;',
```

Comment:
why add the display:none just curious?

Reply:
I actually only added the comma, so it shows up as a change

Reply:
Ah ok, I missed that, thanks.